api/README.V1.md

```diff
@@ -235,6 +235,23 @@ Utilization of hospital resources.
 | currentUsageTotal | integer | Currently used capacity for resource by all patients (COVID + Non-COVID)                                                                                                                                                                |
 | typicalUsageRate  | number  | Typical used capacity rate for resource. This excludes any COVID usage.                                                                                                                                                                 |
 
+### Metrics
+Calculated metrics data based on known actuals.
+
+| name           | type   | description                                                                       |
+|----------------|--------|-----------------------------------------------------------------------------------|
+| testPositivity | number | Ratio of people who test positive calculated using a 7 day rolling average.       |
+| caseDensity    | number | The number of cases per 100k population calculated using a 7 day rolling average. |
+
+### MetricsTimeseriesRow
```

Comment:
Minor: The fact that both of these classes essentially have the same documentation kind of indicates to me that we probably don't need both.

Reply:
Yeah I go back and forth there.  I think the cost of having it is pretty small, but definitely a bit awkward